wagyu/cli/ethereum.rs

```diff
@@ -613,22 +625,24 @@ impl CLI for EthereumCLI {
     fn print(options: Self::Options) -> Result<(), CLIError> {
         fn output<N: EthereumNetwork, W: EthereumWordlist>(options: EthereumOptions) -> Result<(), CLIError> {
             let wallets = match options.subcommand.as_ref().map(String::as_str) {
-                Some("hd") => {
-                    let path = options.to_derivation_path(true).unwrap();
-                    (0..options.count)
-                        .flat_map(|_| {
-                            match EthereumWallet::new_hd::<N, W, _>(
-                                &mut StdRng::from_entropy(),
-                                options.word_count,
-                                options.password.as_ref().map(String::as_str),
-                                &path,
-                            ) {
+                Some("hd") => (0..options.count)
+                    .flat_map(|_| {
+                        let mut rng = StdRng::from_entropy();
+                        let mnemonic = EthereumMnemonic::<N, W>::new_with_count(&mut rng, options.word_count).unwrap();
```

Comment:
Shouldn't this be derived using `new_hd`?

Reply:
> Shouldn't this be derived using `new_hd`?

I was thinking that mnemonic should be common for all indices. Should it be different for all addresses?

Reply:
On the issue user was asking about one mnemonic with different addresses. 
>I have an ethereum mnemonic and am trying to generate 10 addresses using BIP44. What's the best way to do this?

Reply:
Right but using new_hd as originally written does the same thing, right?

Reply:
Howard, no. We need *one* mnemonic for HD wallet, right? So `new_hd` doesn't do that.  
Because `EthereumMnemonic` generates new mnemonic every time with a different range of number. And therefore with a different set of words. So, we need a `new_hd` function that accepts a common mnemonic for generating addresses with this mnemonic. Please, correct me if I'm wrong

Reply:
Here `new_hd` generates a new mnemonic:
https://github.com/AleoHQ/wagyu/blob/8164a7acb9750fa1b0c0bd0397cfea9241fcd0a5/wagyu/cli/ethereum.rs#L67
And then mnemonic in its turn generates new range of numbers:
https://github.com/AleoHQ/wagyu/blob/8164a7acb9750fa1b0c0bd0397cfea9241fcd0a5/ethereum/src/mnemonic.rs#L44

Reply:
I disagree with the proposed change. `new_hd` is supposed to sample a randomly generated wallet (as it is on `master`). You then pass into `from_mnemonic` the HD wallet's mnemonic to fetch subsequent indices. Additionally, this functionality should be achievable in the HD wallet's implementation on its own.

Reply:
I made functionality with `--index` and `--indices` options for `hd` command. If that correct I will add functionality for `import hd` command

Reply:
I added `--indices` option for `import hd` command